.travis.yml

```diff
@@ -68,6 +68,8 @@ jobs:
       script: ./ciRunSbt.sh clean ui/slow:test
     - name: "Management tests"
       script: ./ciRunSbt.sh clean management/it:test
+    - name: "Scala 2.11-2.12 compile"
+      script: CROSS_BUILD=true ./ciRunSbt.sh clean compile
```

Comment:
test:compile?
Can we make it run only if "original" CROSS_BUILD==false?

Reply:
I think that @mproch want to have both `compile:compile` and `test:compile`. Am I right?

Reply:
fixed